Bio_stats_assignment.py

```diff
@@ -62,7 +62,7 @@ def num_older_than(
     today_date: datetime = datetime.today()  # O(1) generate todays date
 
     if "PatientDateOfBirth" not in patient_data.keys():  # O(1)
-        return "No information found regarding Age"  # O(1)
+        raise KeyError(f"No information found regarding Age")  # O(1)
```

Comment:
@patrickkwang  Professor I have changed this from returning strings to raising errors 
Please let me know if there are any other changes required 
Thank you

Reply:
These changes have been made through out the python script (in other places ) and the test file